Add functionality to project

David-Kasilia opened the pull request:
## **To Highlight**
In this project, I  built a simple to do list with webpack and es6  Below is a list of what was implemented:

1. Setup the project by creating a new directory and initializing it with npm using the official webpack documentation to set up webpack.
2. Add the HTML to index.html in src folder.
3. Worked on styling the website using style.css.
4. Created modules for the project and imported them to index.js.
5. Add remove task.js to add and remove tasks from  the list.
6. Create populate task list .js to populate the UI for the task list
7. fix linters errors.
8. test the website by adding and removing tasks.

David-Kasilia committed:
Add functionality to project

David-Kasilia committed:
Fix linters errors

1GORDON commented on src/index.js:
```diff
@@ -1,11 +1,10 @@
 import './style.css';
```
Great job @David-Kasilia 🤝

- [ ] Kindly import the images correctly into your project. The images are lacking in your design
![imagess](https://user-images.githubusercontent.com/63917725/183873938-d2f561d0-0c97-4150-95e4-9a814ff70ca9.PNG)

David-Kasilia replied:
@1GORDON Hello, I would like to request your help in implementing this because I have tried linking the images and the CSS file again and it still not loading the 9images as required.
Is this because of the file loader dependency for webpack?

1GORDON replied:
Hello @David-Kasilia, Kindly take a look at the [webpack getting started](https://webpack.js.org/guides/getting-started/) documentation. Go through the [asset management](https://webpack.js.org/guides/asset-management/) properly.

David-Kasilia replied:
@1GORDON Thanks alot for your help. I went through it again and used the appropriate method it is now functional.You may confirm that if you would like

1GORDON commented on index.html:
```diff
@@ -31,6 +31,5 @@ <h1 class="title">Today's To Do</h1>
        
     </main>
    
-   
 </body>
 </html>
```
Great job @David-Kasilia 🤝

- [x] You may kindly remove this `index.html` file as it is not being served to development.

David-Kasilia replied:
I have implemented that. Thank you

1GORDON reviewed (changes requested):
##
## Hi @David-Kasilia ✋

Good job so far!

## Highlights!
- Followed Gitflow ✔️ 
- Good commit messages ✔️
- Linters are passing ✔️
- Descriptive Pull request ✔️
- Professional readme ✔️

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️

- [x] Kindly format your methods in the `addRemovetask.js` file with ES6 syntax

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

David-Kasilia committed:
fix index.html review

David-Kasilia committed:
Fix review on images

David-Kasilia committed:
Fix linter error

David-Kasilia commented:
### **Review Fixes To Highlight**
Fixed the images not loading issue by importing the icons from font awesome icons.
Removed the index.html file that was not needed
Fixed the  ES6 syntax error

Nemwel-Boniface reviewed (approved):
## Hello @David-Kasilia               :wave: ,
I am Nemwel your reviewer for this PR :man_beard:  

Thank you for making the changes requested by a previous reviewer good job! :partying_face: 
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! :partying_face: 🎉 

![](https://media.giphy.com/media/3kuSo744UIPJjcJUEn/giphy.gif)

## Highlights

Great job setting up the correct linters ✔️ 
Professional documentation ✔️ 
Correct workflow ✔️ 
Meaningful commit messages ✔️ 
Met all the milestone requirements ✔️ 

## [Optional] suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me **@Nemwel-Boniface** in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
